python/cudf/cudf/core/dataframe.py

```diff
@@ -842,6 +847,21 @@ def _init_from_list_like(self, data, index=None, columns=None):
         elif len(data) > 0 and isinstance(data[0], pd._libs.interval.Interval):
             data = DataFrame.from_pandas(pd.DataFrame(data))
             self._data = data._data
+        # namedtuple in a list
+        elif (
+            len(data) > 0
+            and columns is None
+            and isinstance(data[0], tuple)
+            and hasattr(data[0], "_fields")
+        ):
+            # pandas behavior is to use the fields from the first tuple
+            # as the column names
+            columns = data[0]._fields
+            values = itertools.zip_longest(*data)
+            data = dict(zip(columns, values))
+            self._init_from_dict_like(
+                data, index=index, nan_as_null=nan_as_null
+            )
```

Comment:
```suggestion
        if (
            len(data) > 0
            and columns is None
            and isinstance(data[0], tuple)
            and hasattr(data[0], "_fields")
        ):
            # pandas behavior is to use the fields from the first tuple
            # as the column names
            columns = data[0]._fields
```

Will just doing this work?

Reply:
I was curious about this too; since this code path just assigns `self.columns = columns` I wasn't sure if it was okay if `columns` was a tuple or needed to be a different type

Reply:
Okay this looks to work and produces a column Index too so I changed it to this.